ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -50,7 +51,7 @@ function mapStateToProps(state) {
   return {
     provider: state.metamask.provider,
     shouldShowTestNetworks: getShowTestNetworks(state),
-    frequentRpcListDetail: state.metamask.frequentRpcListDetail || [],
+    networkConfigurations: state.metamask.networkConfigurations || [],
```

Comment:
I'm not entirely sure why we need a fallback here, but it certainly shouldn't be an array. Probably best to either remove it, or default to `{}`

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/ef2df393c9838e83875c78b10cc411b0cdb6c76c. No fall back necessary!